File: php/fizzBuzz.php
<?php
function fizzBuzz($n) {
  for ($i = 1; $i <= $n; $i++) {
    if ($i % 5 === 0 && $i % 3 === 0) {
      print_r('FizzBuzz' . "\n");
    } elseif ($i % 5 === 0) {
      print_r('Buzz' . "\n");
    } elseif ($i % 3 === 0) {
      print_r('Fizz' . "\n");
    } else {
      print_r($i . "\n");
    }
  }
}

print_r(fizzBuzz(40));

File: python/fizz_buzz.py
def fizz_buzz(n):
  i = 0;
  while i < n:
    i += 1;
    if i % 5 == 0 and i % 3 == 0:
      print('FizzBuzz');
    elif i % 5 == 0:
      print('Buzz');
    elif i % 3 == 0:
      print('Fizz');
    else:
      print(i);

fizz_buzz(40);